invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/ItemDetails/state/actions.js

```diff
@@ -1,27 +1,69 @@
-import { IS_LOADING, SUCCESS, HAS_ERROR } from './types';
-import { item } from '../../../../common/api';
-import { sendErrorNotification } from '../../../../common/components/Notifications';
+import { ES_DELAY } from '../../../../common/config';
+import {
+  IS_LOADING,
+  SUCCESS,
+  HAS_ERROR,
+  DELETE_IS_LOADING,
+  DELETE_SUCCESS,
+  DELETE_HAS_ERROR,
+} from './types';
+import { item as itemApi } from '../../../../common/api';
+import { BackOfficeURLS } from '../../../../common/urls';
+import {
+  sendErrorNotification,
+  sendSuccessNotification,
+} from '../../../../common/components/Notifications';
+import history from '../../../../history';
 
 export const fetchItemDetails = itemPid => {
   return async dispatch => {
     dispatch({
       type: IS_LOADING,
     });
 
-    await item
-      .get(itemPid)
-      .then(response => {
-        dispatch({
-          type: SUCCESS,
-          payload: response.data,
-        });
-      })
-      .catch(error => {
+    try {
```

Comment:
@ntarocco, replying to your previous comments, I have placed the try catch inside the actions, I think that if we move the try/catch to the `api` we will need something like @zzacharo approach, your thougths?